docs/src/database_import.md

```diff
@@ -82,6 +82,10 @@ performed in 4 steps:
   database via the `postImportApplicationSQL` parameter
 - execution of `ANALYZE VERBOSE` on the imported database
 
+In the below figure, a single PostgreSQL cluster with N databases is imported
+into separate CloudNativePG clusters, each using a microservice import
+for one of the N source databases.
+
```

Comment:
I've added this paragraph because in the image the source cluster has the legend "MONOLITH", so there is potential for confusion.
Can the image be corrected?